### ISSUE TITLE
Lost image after saving v1.9.77

### ISSUE THREAD

**Issue by frostblazergit** [gildas-lormeau/SingleFile#168]
**Describe the bug**
After saving, the image is lost. Try this URL:  
https://kylingit.com/blog/weblogic-xmldecoder-rce%E4%B9%8Brmi%E5%88%A9%E7%94%A8/
Save

**To Reproduce**
1. Go to 'https://kylingit.com/blog/weblogic-xmldecoder-rce%E4%B9%8Brmi%E5%88%A9%E7%94%A8/'
2. Save Page
3. Image loss

**Environment**
 - OS: macOS 10.14.1
 - Browser: Chrome 72.0.3626.119
 - Version: SingleFile 1.9.77

**Comment by gildas-lormeau** [comment]
I'm surprised it was working in the past. For your information, like the previous similar issue (#166), this is due to the fact the image is served with an invalid Content-Type header. Here the value is `false` when trying to download https://blog-1252261399.cos-website.ap-beijing.myqcloud.com/images/b2MmC. It will be fixed very soon.

**Comment by frostblazergit** [comment]
@gildas-lormeau 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

```
Generic binary data (or binary data whose true type is unknown) is application/octet-stream.
```

Can this be set to "application/octet-stream" for this similar situation?

**Comment by gildas-lormeau** [comment]
Yes it can. SingleFile supports this value but not `false`. However, instead of white-listing content-types, I'll blacklist them. The one that I don't want to see in images in `text/html` for example. This happens more often than we think. In a `<img>` tag, if a developer set the `src` attribute value to `""`, the browser will try to display the page itself into the `<img>` tag. Some other extensions (that I won't cite) do the same kind of error by setting the `src` value of `<script>` tags or font sources to "".

**Comment by gildas-lormeau** [comment]
This issue is fixed in the version 1.9.78.

**Comment by frostblazergit** [comment]
The problem was solved in the SingleFile 1.9.79 release. Thanks.

**Comment by gildas-lormeau** [comment]
Thank you also for the feedback @frostblazergit
